cumulusci/cli/org.py

```diff
@@ -319,13 +323,26 @@ def org_list(runtime, plain):
                 [org_days, not org_config.active, org_config.config_name, domain]
             )
             scratch_data.append(row)
+            json_data[row[0]] = {
```

Comment:
This feels a bit fragile to me. I'd suggest making the JSON output independent of the column content and ordering in the table. Too easy to change the latter and break the former down the line.